tests/testsuite_recording/integration/testbase.py

```diff
@@ -56,7 +61,9 @@ def setUp(self):
         response_file = self.get_datafile_filename()
         PersistentObjectStorage().storage_file = response_file
         PersistentObjectStorage().dump_after_store = True
-
+        self.static_tmp = "/tmp/packit_tmp"
```

Comment:
eiiwwww, why?

Reply:
this will prevent parallel local test runs (not a big deal but still... hardcoded temp dir paths are bad, there is a whole class of CWEs for that)